sources/packages/backend/apps/api/src/app.students.module.ts

```diff
@@ -53,6 +53,7 @@ import {
   ScholasticStandingStudentsController,
   ProgramYearControllerService,
   AnnouncementStudentsController,
+  DynamicFormConfigurationStudentsController,
```

Comment:
The dynamic form configurations seem generic enough to be added under the `dynamic-form` endpoint.
It does not seem to expose any data that would require, for instance, different authentication methods for each API client type.

Reply:
Oh yes. I can make this controller to be in same way similar to `DynamicFormController`